Set-EntraAttributeSet (#805)
microsoftgraph/entra-powershell: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAttributeSet.md

---
title: Set-EntraAttributeSet
description: This article provides details on the Set-EntraAttributeSet command.

> ms.service: entra

Reviewer: Remove this line. It is handled by doc repo configuration.

ms.topic: reference
ms.date: 07/19/2024
ms.author: eunicewaweru
ms.reviewer: stevemutungi
manager: CelesteDG
author: msewaweru

external help file: Microsoft.Graph.Entra-Help.xml
Module Name: Microsoft.Graph.Entra
online version:
schema: 2.0.0
---

# Set-EntraAttributeSet

## SYNOPSIS

Updates an existing attribute set.

## SYNTAX

```powershell
Set-EntraAttributeSet 
 -Id <String> 
 [-Description <String>] 
 [-MaxAttributesPerSet <Int32>]
 [<CommonParameters>]
```

## DESCRIPTION

> This `Set-EntraAttributeSet` cmdlet Update a Microsoft Entra ID attribute set object identified by ID. Specify `Id` parameter to Update a Microsoft Entra ID attribute set object.

Reviewer: ```suggestion
The `Set-EntraAttributeSet` cmdlet updates a Microsoft Entra ID attribute set object specified by its ID. Specify `Id` parameter to Update a Microsoft Entra ID attribute set object.
```
